aws-java-sdk-core/src/main/java/com/amazonaws/regions/Regions.java

```diff
@@ -62,6 +64,13 @@ private Regions(String name) {
     public String getName() {
         return name;
     }
+    
+    /**
+     * Descriptive readable name for this region.
+     */
+    public String getReadableName() {

```

Comment:
Can we rename it to `getDescription` and rename the member variable to `description` so that it matches with what we have in endpoints.json?

https://github.com/aws/aws-sdk-java/blob/6927b2eda56da6b0737aff14d1d02775011c4547/aws-java-sdk-core/src/main/resources/com/amazonaws/partitions/endpoints.json#L14

Reply:
Sure thing!